packages/scss/src/components/_grid.scss

```diff
@@ -131,7 +131,7 @@
 
 /* SM - Columns */
 
-@media only screen and (min-width: $sm) {
+@media only screen and (min-width: _get($luTheme, "breakpoints.sm")) {
 	.grid-sm {
 		flex-basis: 0;
```

Comment:
For flex, I can't remember where I saw that, but that was prolly browser-support related.

Regarding browser support, I guess that depends on the application. We (Timmi) should support IE 10 and up.